code/modules/heavy_vehicle/mech_interaction.dm

```diff
@@ -320,7 +320,7 @@
 	if(..() && !istype(loc, /turf/space))
 		if(legs && legs.mech_step_sound)
 			playsound(src.loc,legs.mech_step_sound,40,1)
-		use_cell_power(legs.power_use * CELLRATE)
+			use_cell_power(legs.power_use * CELLRATE)
```

Comment:
Why did you change that? -> That way the mech legs only use power if they have a step sound?